var/spack/repos/builtin/packages/mesa/package.py

```diff
@@ -19,13 +19,16 @@ class Mesa(MesonPackage):
     url = "https://archive.mesa3d.org/mesa-20.2.1.tar.xz"
 
     version("main", tag="main")
-    version("22.3.2", sha256="c15df758a8795f53e57f2a228eb4593c22b16dffd9b38f83901f76cd9533140b")
-    version("22.2.5", sha256="850f063146f8ebb262aec04f666c2c1e5623f2a1987dda24e4361b17b912c73b")
+
     version(
-        "22.1.6",
-        sha256="22ced061eb9adab8ea35368246c1995c09723f3f71653cd5050c5cec376e671a",
+        "23.0.3",
+        sha256="386362a5d80df3b096636b67f340e1ce67b705b44767d5bdd11d2ed1037192d5",
         preferred=True,
```

Comment:
The latest stable release is preferred by default, no need for `preferred=True`

Reply:
thanks..made the changes.